src/app/clusters/door-lock-server/door-lock-delegate.h

```diff
@@ -0,0 +1,121 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/zap-generated/cluster-objects.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace DoorLock {
+
+static constexpr size_t kAliroReaderVerificationKeyMaxSize = 65;
+
+static constexpr size_t kAliroAttributeMaxSize_16 = 16;
+
+static constexpr size_t kProtocolVersionMaxSize = 2;
```

Comment:
the spec says 16[2] so I would assume that the size of the protocol version list is 16 and the protocol version itself has a length of 2. Is that correct @bzbarsky-apple?

Reply:
i fixed the spec - https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/8796